docs/developers/developer-reference/smart-contract-developer-best-practices.md

```diff
@@ -198,16 +198,16 @@ Keep in mind that all the mappers can have as many additional arguments for the
 fn list_per_user_pair(&self, first_addr: &ManagedAddress, second_addr: &ManagedAddress) -> VecMapper<T>;
 ```
 
-Using the correct mapper for your situation can greatly decrease gas costs and complexity, so always remember to carefully evaluate your use-case.  
+Using the correct mapper for your situation can greatly decrease gas costs and complexity, so always remember to carefully evaluate your use-case.
 
 ## VarArgs and MultiResults
 
-The Rust language does not natively support var args. The best you could probably do in native rust is use `Option<T>`. Even so, through some pre-processing of arguments, the Rust framework does provide support for said var args, and even multi results.  
+The Rust language does not natively support var args. The best you could probably do in native rust is use `Option<T>`. Even so, through some pre-processing of arguments, the Rust framework does provide support for said var args, and even multi results.
```

Comment:
The Rust language does not natively support variadic arguments. Smart contracts, on the other hand, have no limitations on accepting a variable number of inputs or producing a variable number of results.
To accommodate this, and to make I/O processing succint, the Rust framework provides a number of so-called multi-value types, that deserialize from multiple inputs and serialize to multiple outputs.
Please note that the same types are used both as arguments and as results. This makes sense especially for places like the callbacks, where the results of the asynchronous call become the inputs of the callback.

Reply:
Replaced the paragraph with yours.